### ISSUE TITLE
PR: sensor error

### ISSUE THREAD

**Issue by Eddy69** [bubbledevteam/diabox#56]
After upgrading to the 20210802 pre-release, I get a sensor error. Normally this kind of error occurs only after changing sensor, and resolves by itself after a few hours (start and first read by Libre non-bt reader). Wonder if this is the case now. Sensor has 9 hours left.
Libre 2/Android 10/Samsung S10+

**Comment by Eddy69** [comment]
Update: placed new sensor (started as described above), cleared bt cache and rebooted phone. Now works like earlier releases.
I hate the fixed larger font though, feels like an app from 2013 or so ...

**Comment by bubbledevteam** [comment]
> After upgrading to the 20210802 pre-release, I get a sensor error. Normally this kind of error occurs only after changing sensor, and resolves by itself after a few hours (start and first read by Libre non-bt reader). Wonder if this is the case now. Sensor has 9 hours left.
> Libre 2/Android 10/Samsung S10+

could you pls share a log to :diaboxapp@gmail.com?

**Comment by bubbledevteam** [comment]
> Update: placed new sensor (started as described above), cleared bt cache and rebooted phone. Now works like earlier releases.
> I hate the fixed larger font though, feels like an app from 2013 or so ...

could you pls show me a pic about this ?

**Comment by Eddy69** [comment]
![Screenshot_20210804-075329_DiaBox](https://user-images.githubusercontent.com/31344113/128132543-6e95094a-7f1e-4792-bd86-b4b6263c3f0e.jpg)
In the Android settings, the font size is set to smallest.

**Comment by Eddy69** [comment]
> > After upgrading to the 20210802 pre-release, I get a sensor error. Normally this kind of error occurs only after changing sensor, and resolves by itself after a few hours (start and first read by Libre non-bt reader). Wonder if this is the case now. Sensor has 9 hours left.
> > Libre 2/Android 10/Samsung S10+
> 
> could you pls share a log to :[diaboxapp@gmail.com](mailto:diaboxapp@gmail.com)?

Done.

**Comment by bubbledevteam** [comment]
> ![Screenshot_20210804-075329_DiaBox](https://user-images.githubusercontent.com/31344113/128132543-6e95094a-7f1e-4792-bd86-b4b6263c3f0e.jpg)
> In the Android settings, the font size is set to smallest.

so you want it smaller？

**Comment by Eddy69** [comment]
Yes, please. Scaling according to Android settings would be best, I guess 🤔
